images/singleuser/Dockerfile

```diff
@@ -80,42 +88,49 @@ RUN apt-get install --yes \
     libzmq3-dev \
     libreadline-dev \
     # For R's mysql
-    libmariadb-client-lgpl-dev \
+    libmariadb-dev \
     # For R's curl
     libcurl4-openssl-dev \
     # For R's devtools
     libssl-dev \
     # For PDFs and stuff
     pandoc \
-    texlive-xetex
+    texlive-xetex > /dev/null
+
+# Create user owned R libs dir
+# This lets users temporarily install packages
+ENV R_LIBS_USER /srv/r
+RUN install -d -o ${NB_USER} -g ${NB_USER} ${R_LIBS_USER}
+
+# Create venv directory, and let users install into it
+ENV VENV_DIR /srv/paws
+RUN install -d -o ${NB_USER} -g ${NB_USER} ${VENV_DIR}
 
-RUN mkdir -p /srv/paws && chown ${NB_USER}:${NB_USER} /srv/paws
 ENV PATH=/srv/paws/pwb:/srv/paws/bin:/srv/paws:$PATH
 
 USER ${NB_USER}
-RUN python3.6 -m venv /srv/paws
+RUN python3 -m venv /srv/paws
 
 # Install base notebook packages
 RUN pip install --no-cache-dir \
     jupyterhub==1.1.0 \
```

Comment:
any reason to keep 1.1.0 here?

Reply:
Since the chart itself is running z2jh 0.9, which is jupyterhub 1.1. Once the chart is upgraded, this can be too